macOS/DuckDuckGo/Common/Utilities/UserDefaultsWrapper.swift

```diff
@@ -24,7 +24,6 @@ import Foundation
 extension UserDefaults {
     /// The app group's shared UserDefaults
     static let netP = UserDefaults(suiteName: Bundle.main.appGroup(bundle: .netP))!
-    static let dbp = UserDefaults(suiteName: Bundle.main.appGroup(bundle: .dbp))!
```

Comment:
This wasn't being used anywhere as far as I could tell, and was causing compilation errors due to ambiguity when trying to access the `.dbp` UserDefaults, since we also define an extension like this inside DBP Core.